### PR TITLE
story book formly stepper

### PR DESCRIPTION
**Pull request by shayan-roshan**
sb formly stepper demo

## Type of Change (Select One and Apply Github Label)

- [ ] Bug fix (non-breaking change which fixes an issue) -> Apply bugfix label
- [ ] New feature (non-breaking change which adds functionality) -> Apply enhancement label
- [ ] Breaking change (fix or feature that would cause existing functionality to change) -> Apply breaking label

## Which browsers have you tested?
- [ ] Internet Explorer 11
- [ ] Edge
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

## Checklist:

- [ ] I have read the (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md] document.
- [ ] My code passes the automated linter.
- [ ] This code has been reviewed by another team member and passes the reviewer checklist found in (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md]
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code is 508 compliant as tested by AMP and JAWS
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/documentation/src/lib/storybook/formly/formly-stepper/formly-stepper.stories.ts [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        FormlyStepperIntroductionModule,
+        RouterTestingModule,
+        FormlyStepperAdvancedModule,
+        UswdsStepperModule,
+      ],
+      providers: [SdsStepper],
+    }),
+  ],
+  argTypes: {
+    label: templateOptions,
```
Comment (cwolf10) [inline libs/documentation/src/lib/storybook/formly/formly-stepper/formly-stepper.stories.ts:47]: Can remove these argTypes

File: libs/documentation/src/lib/storybook/formly/formly-stepper/formly-stepper.stories.ts [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        'storybook/formly/formly-stepper/formly-stepper-basic/formly-stepper-basic-uneven.component.html',
+        'html',
+        false
+      ),
+    ]
+  ),
+  stackblitzLink: generateStackblitzLink('formly-stepper', 'basic'),
+};
+
+export const USWDS: Story = (args) => ({
```
Comment (cwolf10) [inline libs/documentation/src/lib/storybook/formly/formly-stepper/formly-stepper.stories.ts:92]: Stackblitz fails:
ERROR
Error: R3InjectorError(AppModule)[Router -> Router -> Router]:
NullInjectorError: No provider for Router!

File: libs/documentation/src/lib/storybook/formly/formly-stepper/formly-stepper.stories.ts [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+  template: '<stepper-advanced-demo></stepper-advanced-demo>',
+  props: args,
+});
+Advanced.parameters = {
+  controls: {
+    disabled: true,
+    hideNoControlsWarning: true,
+  },
+  actions: { disabled: true },
+  preview: generateConfig(
```
Comment (cwolf10) [inline libs/documentation/src/lib/storybook/formly/formly-stepper/formly-stepper.stories.ts:132]: Stackblitz is failing to initialize:
Error in src/app/formly-stepper-advanced.component.html (49:57)
Type '{ key: string; templateOptions: { placeholder: string; }; fieldGroup: ({ key: string; fieldGroupClassName: string; fieldGroup: ({ className: string; key: string; type: string; templateOptions: { ...; }; } | { ...; })[]; className?: undefined; type?: undefined; templateOptions?: undefined; modelOptions?: undefined; }...' is not assignable to type 'FormlyFieldConfig'.
Types of property 'fieldGroup' are incompatible.
Type '({ key: string; fieldGroupClassName: string; fieldGroup: ({ className: string; key: string; type: string; templateOptions: { label: string; hideOptional: boolean; options: { label: string; value: string; }[]; required: boolean; showError?: undefined; }; } | { ...; })[]; className?: undefined; type?: undefined; templ...' is not assignable to type 'FormlyFieldConfig[]'.
Type '{ key: string; fieldGroupClassName: string; fieldGroup: ({ className: string; key: string; type: string; templateOptions: { label: string; hideOptional: boolean; options: { label: string; value: string; }[]; required: boolean; showError?: undefined; }; } | { ...; })[]; className?: undefined; type?: undefined; templa...' is not assignable to type 'FormlyFieldConfig'.
Type '{ className: string; type: string; key: string; templateOptions: { label: string; required: boolean; showError: boolean; }; modelOptions: { updateOn: string; }; fieldGroupClassName?: undefined; fieldGroup?: undefined; }' is not assignable to type 'FormlyFieldConfig'.
The types of 'modelOptions.updateOn' are incompatible between these types.
Type 'string' is not assignable to type '"blur" | "change" | "submit"'.

**Review by cwolf10 — CHANGES_REQUESTED** [review]

**Comment by cwolf10** [comment]
Looks like the nav buttons introduced some lint issues. Also, did you manage to get the stackblitzes updated? It looks like the only update pushed since my last comment was the update to the nav buttons.

**Comment by shayan-roshan** [comment]
this is not a final version of the branch that is good to be merged, i just made a commit to save some changes. 
I am still waiting on the final changes that need to be made on the pill buttons icons and then I will update the stepper.

**Comment by cwolf10** [comment]
All good, just wanted to make sure I wasn't blocking getting this merged.

**Pull request closed without merging**